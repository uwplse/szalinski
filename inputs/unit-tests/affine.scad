cube();
translate([1, 2, 3]) {
    rotate([90, 60, 30]) {
        scale([4, 5, 6]) {
            cube();
        }
    }
}