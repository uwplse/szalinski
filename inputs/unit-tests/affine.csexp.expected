(Union
(Cube (Vec3 1 1 1) false)
(Trans (Vec3 1 2 3)
  (Rotate (Vec3 90 59.999953729744036 29.999982919691146)
    (Scale (Vec3 4 5 6)
      (Cube (Vec3 1 1 1) false)))))